### PR TITLE
Add types for pendo feedback loginAndRedirect

### PR DESCRIPTION
**Pull request by jquintozamora**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://support.pendo.io/hc/en-us/articles/360060085212-How-to-give-your-visitors-access-to-Feedback#1Portalviewfullscreendefault

### CODE REVIEW

File: types/pendo-io-browser/index.d.ts [hunk 1]
```diff
@@ -83,11 +83,21 @@ declare namespace pendo {
...
         // Other
         validateInstall(): void;
         dom(input: any): HTMLElement; // TODO
     }
 
+    interface FeedbackOptions {
+        anchor: HTMLElement;
+    }
+    interface Feedback {
+        loginAndRedirect: (options?: FeedbackOptions) => void;
```
Comment (tommy-josepovic) [inline types/pendo-io-browser/index.d.ts:98]: Nit: To stay consistent with the rest of the file.
```suggestion
        loginAndRedirect(options?: FeedbackOptions): void;
```

**Review by tjosepo — APPROVED** [review]

**Comment by jquintozamora** [comment]
Ready to merge

**Comment by joseq-concentra** [comment]
@tommy-josepovic I've committed your suggested changes, thanks!

Please could you submit your review again, appreciated.

**Review by tjosepo — APPROVED** [review]

**Comment by jquintozamora** [comment]
Ready to merge

**Pull request merged**